apps/confidential/confidential.go

```diff
@@ -756,18 +771,19 @@ type AcquireOnBehalfOfOption interface {
 // AcquireTokenOnBehalfOf acquires a security token for an app using middle tier apps access token.
 // Refer https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow.
 //
-// Options: [WithClaims], [WithTenantID]
+// Options: [WithClaims], [WithTenantID], [withExtraBodyParameters]
```

Comment:
1. you can't link to this symbol's docs because this symbol isn't documented, because it isn't exported
2. there's no reason to document this option anyway because applications can't use it, because it isn't exported
```suggestion
// Options: [WithClaims], [WithTenantID]
```